main/conceptual/ertp/README.md

```diff
@@ -216,21 +216,21 @@ Things end up with your Quatloos `purse` having 2 Quatloos (7 - 5), Alice's Quat
 in it, and the 5 Quatloos `payment` consumed when the transfer happened.
 
 The `E()` notation is a local "bridge" that lets you invoke methods on remote objects.
-It takes a local representative (a [presence](/glossary/#presence)) for a remote object
+It takes a local representative (a [presence](../../glossary/#presence)) for a remote object
 as an argument and sends messages to the remote object. This is explained in more detail at the
-[`E()` section in the Distributed JavaScript page](/guides/js-programming/eventual-send.md).
+[`E()` section in the Distributed JavaScript page](../js-programming/eventual-send.md).
 
-## Creating and using non-fungible assets
+## Creating and Using Non-Fungible Assets
 
 Say you own the Agoric Theatre and want to sell tickets to seats for a play. Tickets are non-fungible assets, 
 as they refer to a specific seat for a specific show at a specific time and date. It matters to
 buyers which ticket they get.
 
 The Agoric Theatre has 1114 seats numbered `1` to `1114`.
 An object representing a valid ticket has the properties:
-- `seat`: A number
-- `show`: A string describing the show
-- `start`: A string representing a [time/date in ISO format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)
+- `seat`: A number.
```

Comment:
hm. periods for things that are not sentences?

Reply:
I'm following the MSFT guide for those because I'm used to it & it "looks right" to me. The rule is: descriptions like that, even when they're sentence fragments, start with a capital letter and end with a period.